cmd/unbuild.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"github.com/qlik-oss/corectl/internal"
+	"github.com/spf13/cobra"
+)
+
+var unbuildCmd = withLocalFlags(&cobra.Command{
+	Use:     "unbuild",
+	Args:    cobra.ExactArgs(0),
+	Short:   "Split upp an existing app into separate entities",
+	Long:    "Split upp an existing app into separate entities",
```

Comment:
We should probably add more info to the `Long` description, e.g. what entities we do export.